src/crystal_lib/parser.cr

```diff
@@ -4,8 +4,13 @@ require "clang/clang-c/Index"
 class CrystalLib::Parser
   getter nodes
 
-  def self.parse(source, flags = [] of String)
-    parser = Parser.new(source, flags)
+  enum Option
```

Comment:
Maybe this should be a flag enum and `Parser#@options` just be of type `Option`?

Then, instead of `@options.includes?(Option::ImportFullComments)` you could just do `@options.import_full_comments?`.

Reply:
Makes sense, going to update it (my Crystal is a bit rusty :P).